### DISCUSSION TITLE
Multiple layout pdf from single plugin

### DISCUSSION THREAD

**Discussion by Shailendra1512** [dita-ot/dita-ot#4169] [Q&A]
Hello Everyone,

I am facing one issue, my client gave me a requirement where I need to create different layout (according publication type) from single Dita custom plugin (transtype should be same). I have one metadata file where I can found different publication entry.

So far i create custom plugin by which pdf is generated.

Dita-ot version: 3.7.4

When i found field 
@name = "FDITAOTTRANSTYPE"  and value is fixedpdf then transtype =  fixedpdf
nested condition inside it:
  @name = "FISHOUTPUTFORMATNAME" then xslt script run accordingly.

**Metadata.met**
```
<objects>
    <object ishref="VOUTPUTFORMATPDFL4AASIL4AAPDF" type="OutputFormat"
        outputformatref="6314">
        <fields>
            <field name="NAME" level="none" xml:space="preserve">SI_PDF</field>
            <field name="FISHOUTPUTFORMATNAME" level="none" xml:space="preserve">SI_PDF</field>
            <field name="FISHRESOLUTIONS" level="none" xml:space="preserve">High, Low</field>
            <field name="FISHKEEPDTDSYSTEMID" level="none" xml:space="preserve">Yes</field>
            <field name="FISHOUTPUTEDT" level="none" xml:space="preserve">PDF</field>
            <field name="FISHSINGLEFILE" level="none" xml:space="preserve">Yes</field>
            <field name="FISHCLEANUP" level="none" xml:space="preserve">No</field>
            <field name="FDITAOTTRANSTYPE" level="none" xml:space="preserve">fixedpdf</field> 
</fields>
</object>
</objects>
```

Kindly suggest how to achieve this.
Mail id: rananeeraj15dec@gmail.com

Thank you.

**Comment by stefan-jung** [comment]
It is possible to do that. You can also reuse stylesheets between the transformations and override them multiple times. You should read the documentation about creating pdf plugins in the DITA-OT docs and buy the book DITA for Print.

**Comment by jelovirt** [comment]
Edited the original post where the contents of the metadata was not visible because it was not inside a code block (three backticks). Remember to preview your posts to make sure all your content is visible.

**Reply by Shailendra1512** [reply]
Thanks @jelovirt
